crates/uv-settings/src/settings.rs

```diff
@@ -449,31 +449,352 @@ pub struct ResolverInstallerOptions {
+    ///
+    /// Supported formats:
+    /// - `3.10` looks for an installed Python 3.10 using `py --list-paths` on Windows, or
+    ///   `python3.10` on Linux and macOS.
+    /// - `python3.10` or `python.exe` looks for a binary with the given name in `PATH`.
+    /// - `/home/ferris/.local/bin/python3.10` uses the exact Python at the given path.
+    #[option(
+        default = "None",
+        value_type = "str",
+        example = r#"
+            python = "3.10"
+        "#
+    )]
     pub python: Option<String>,
+    /// Install packages into the system Python environment.
+    ///
+    /// By default, uv installs into the virtual environment in the current working directory or
+    /// any parent directory. The `--system` option instructs uv to instead use the first Python
+    /// found in the system `PATH`.
+    ///
+    /// WARNING: `--system` is intended for use in continuous integration (CI) environments and
+    /// should be used with caution, as it can modify the system Python installation.
+    #[option(
+        default = "false",
+        value_type = "bool",
+        example = r#"
+            system = true
+        "#
+    )]
     pub system: Option<bool>,
+    /// Allow uv to modify an `EXTERNALLY-MANAGED` Python installation.
+    ///
+    /// WARNING: `--break-system-packages` is intended for use in continuous integration (CI)
+    /// environments, when installing into Python installations that are managed by an external
+    /// package manager, like `apt`. It should be used with caution, as such Python installations
+    /// explicitly recommend against modifications by other package managers (like uv or pip).
+    #[option(
+        default = "false",
+        value_type = "bool",
+        example = r#"
+            break-system-packages = true
+        "#
+    )]
     pub break_system_packages: Option<bool>,
+    /// Install packages into the specified directory, rather than into the virtual or system Python
+    /// environment. The packages will be installed at the top-level of the directory.
+    #[option(
+        default = "None",
+        value_type = "str",
+        example = r#"
+            target = "./target"
+        "#
+    )]
     pub target: Option<PathBuf>,
+    /// Install packages into `lib`, `bin`, and other top-level folders under the specified
+    /// directory, as if a virtual environment were present at that location.
+    ///
+    /// In general, prefer the use of `--python` to install into an alternate environment, as
+    /// scripts and other artifacts installed via `--prefix` will reference the installing
+    /// interpreter, rather than any interpreter added to the `--prefix` directory, rendering them
+    /// non-portable.
+    #[option(
+        default = "None",
+        value_type = "str",
+        example = r#"
+            prefix = "./prefix"
+        "#
+    )]
     pub prefix: Option<PathBuf>,
+    /// The URL of the Python package index (by default: <https://pypi.org/simple>).
```

Comment:
Does this URL format work well in mkdocs? (Sorry for not checking)

Reply:
Turns out it does.